### ISSUE TITLE
[Bug]: Multiple definitions exist for class 'Builder'

### ISSUE THREAD

**Issue by ryanmortier** [laravel-idea/plugin#1096]
### Bug description

Hello,

I'm suddenly getting these warnings all over my code base. Wondering if this is an issue with Laravel Idea or something else?

![CleanShot 2024-11-06 at 11 49 12](https://github.com/user-attachments/assets/d5bc349f-c883-43b1-b9b2-e9173f224180)

### Plugin version

8.4.0.241

### Operating system

MacOS

**Comment by adelf** [comment]
Do you have "kirschbaum-development/eloquent-power-joins" package installed?

If not, could you show all 4 sources of the Builder class(I see there are 4 places in your screenshot)?

**Comment by ryanmortier** [comment]
I do not have that directly installed but I do have `filament/support` installed which has that as a dependency.

These are the 4:

```
vendor/kirschbaum-development/eloquent-power-joins/.stubs.php
vendor/_laravel_idea/_ide_helper_macro_static.php
vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php
vendor/_laravel_idea/_ide_helper_macro.php
```

**Comment by adelf** [comment]
It's fixed by Laravel Idea 9.0. Please update.
